tests/functional/pod_and_daemons/test_csi_logs_rotation.py

```diff
@@ -0,0 +1,178 @@
+import time
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import BaseTest
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.constants import OPENSHIFT_STORAGE_NAMESPACE
+from ocs_ci.framework.pytest_customization.marks import brown_squad, tier2
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+
+log = logging.getLogger(__name__)
+
+WAIT_FOR_ROTATION_TIME = 1200  # seconds
+SLEEP_BETWEEN_TRIES = 300  # seconds
+
+
+@brown_squad
+@tier2
+class TestPodsCsiLogRotation(BaseTest):
+    def logs_were_rotated(
+        self, pod_obj, gz_logs_num, current_log_file_size, logs_dir, log_file_name
+    ):
+        """
+        Gets csi pod log files details
```

Comment:
Function name and description is confusing, Looks like function checks for log rotation is successful, so func name could be check_for_log_rotation_successful() and to add appropriate description.
If this function is generic can be moved to some common library file than the test script.

Reply:
@ramkiperiy The function name was renames according to your suggestion. As for the function location - I prefer to keep it as part of the class since this is a spasific and not generic rotation of csi pods only.